templater/jobs/utils/utils.go

```diff
@@ -75,6 +83,32 @@ func UnmarshalJobs(jobDir string) (map[string]types.JobConfig, error) {
 
 				jobList[releaseBranchBasedFileName] = jobConfig
 			}
+		} else if strings.Contains(fileName, "1.X") {
```

Comment:
Agreed, the placeholder inside the minimal config would still be `1.X` though.

Reply:
I realize that this will only replace the `golangVersion` value in the template if it's present, but it'd be nice to only trigger the templating on golang-specific files, so that if we want to implement this pattern for other dependancies in the future we don't accidentally end up clobbering shared values